dev/service-registry/client-core/src/main/scala/com/lightbend/lagom/devmode/internal/registry/LagomDevModeServiceDiscovery.scala

```diff
@@ -13,10 +13,10 @@ import akka.discovery.Discovery
 import akka.discovery.Lookup
 import akka.discovery.ServiceDiscovery
 
-import scala.concurrent.duration.FiniteDuration
 import scala.concurrent.ExecutionContext
 import scala.concurrent.Future
 import scala.concurrent.Promise
+import scala.concurrent.duration.FiniteDuration
 
 private[lagom] class DevModeServiceDiscovery(system: ActorSystem) extends ServiceDiscovery {
```

Comment:
Only file renamed. Need to rename class too. 😉 
```suggestion
private[lagom] class LagomDevModeServiceDiscovery(system: ActorSystem) extends ServiceDiscovery {
```

Reply:
nice too :)

Reply:
@ihostage after looking on the code the dependencies of DevModeServiceDiscovery are as follows:
- com.lightbend.lagom.internal.javadsl.registry.ServiceRegistryModule
- com.lightbend.lagom.scaladsl.devmode.LagomDevModeComponents
i didn't change the name of DevModeServiceDiscovery to LagomDevModeServiceDiscovery because i was preventing too much changes. the dependencies are not that much and can be changed. what do you think?

Reply:
I think that we should be renaming the file and class together. Both or nothing.  :smile: